docs/developer-docs/integrations/sns/get-sns/local-testing.md

```diff
@@ -134,54 +241,54 @@ If the proposal is rejected, the dapp canisters' controllers are automatically s
-maximum ICP that the sale accepts.
+Note the sale start time. Wait until then, then hit refresh. You should now see an interface to buy SNS tokens.
+Buy some tokens. The sale will be complete when either the maximum investment has been reached or the sale end time is reached.
+If you use the default SNS configuration you can buy all 50 ICP.
+This is convenient for testing but in a real SNS you may wish to limit the stake so that no investor has excessive influence over the project.
 
-#### 11. Repeatedly test upgrading the dapp canister(s) and adding more canisters.
+You can then check the neurons of the SNS:
+
+``` bash
+dfx canister call sns_governance list_neurons '(record {limit= 20})'
+```
+
+#### 10. Repeatedly test upgrading the dapp canister(s) and adding more canisters.
 You might want to repeatedly test that you can upgrade the dapp at all stages.
-As already explained in [Step7](#7-test-upgrading-the-dapp-canisters-by-sns-proposal),
+As already explained in [Step 6](#6-test-upgrading-the-dapp-canisters-by-sns-proposal),
 you might want to test this before the decentralization sale and you might also want to test this
 during and after the decentralization sale.
 To do so, you can repeat the instructions from
-[Step 7](#7-test-upgrading-the-dapp-canisters-by-sns-proposal).
+[Step 6](#6-test-upgrading-the-dapp-canisters-by-sns-proposal).
 
 You can also test adding new dapp canisters under the SNS control at different stages,
-by additional SNS proposals as explained in [Step 6](#6-register-the-dapp-in-the-sns).
+by additional SNS proposals as explained in [here](https://github.com/dfinity/sns-quill#submit-a-proposal).
+For example, you could make a proposal to set the minimum neuron stake like this:
+
+``` bash
+$ sns-quill --pem-file ~/.config/dfx/identity/$(dfx identity whoami)/identity.pem --canister-ids-file canister_ids.json make-proposal $DEVELOPER_NEURON_ID --proposal '(record { title="Smaller minimal neuron stake limit"; url=""; summary="update minimum neuron stake limit"; action=opt variant{ NervousSystemParameters=record{neuron_minimum_stake_e8s=opt 100000 : opt nat64; } }})'
+```
 
-#### 12. Test launched SNS.
+#### 11. Test launched SNS.
 If the sale has been successful, you can test if the SNS is now fully functional.
 For example, if some initial neurons were set up with dissolve delay zero, you can dissolve them.
-To do so, use your `sns-quill` principal `identityDevNeuron`
-and learn what command to use `here` <!-- TODO: SNS quill documentation to use this neuron mnmt
-command.-->
+You can do that in the NNS dapp.
 
 Also, you may now test other SNS proposal types that did not work while the SNS was still
 in pre-decentralization-sale mode.
 For example, you can submit and vote for a proposal to change some of the SNS parameters
-by following the instructions in [Step](#7-test-upgrading-the-dapp-canisters-by-sns-proposal).
+by following the instructions in [Step 6](#6-test-upgrading-the-dapp-canisters-by-sns-proposal).
```

Comment:
I think the link does not match an example: the link points to proposals upgrading the dapp, while the text talks about changing the SNS parameters.
I suggest removign the link and adding the example from Step 10 instead.